lib/dpp/CachedStateRepositoryDecorator.js

```diff
@@ -55,6 +55,18 @@ class CachedStateRepositoryDecorator {
     return this.stateRepository.fetchPublicKeyIdentityId(publicKeyHash);
```

Comment:
Do we still need it?

Reply:
@shumkov I would like to leave it to keep tests working. After we merge it, we update tests, and then remove this method as a separate PR. This will save us some nerves.